rustls-platform-verifier/src/verification/mod.rs

```diff
@@ -82,8 +82,11 @@ fn invalid_certificate(reason: impl Into<String>) -> rustls::Error {
 /// - id-kp-serverAuth
 // TODO: Chromium also allows for `OID_ANY_EKU` on Android.
 #[cfg(target_os = "windows")]
+// XXX: Windows requires that we NUL terminate EKU strings and we want to make sure that only the
```

Comment:
I think this sentence is missing a word. Otherwise LGTM!

Reply:
Oops, good catch. I'll fix this when migrating to `windows_sys`.